index.html

```diff
@@ -1,145 +1,177 @@
+        });
+
+        function load(rootUrl, fileName, backgroundFileName) {
+            if (engine) {
+                alpha = camera.alpha;
+                beta = camera.beta;
+                radius = camera.radius;
+
+                engine.dispose();
+            }
+
+            engine = new BABYLON.Engine(canvas, true);
+            engine.enableOfflineSupport = false;
+
+            BABYLON.GLTFFileLoader.IncrementalLoading = false;
+
+            BABYLON.SceneLoader.ShowLoadingScreen = false;
+
+            BABYLON.SceneLoader.Load(rootUrl, fileName, engine, function (scene) {
+                scene.clearColor = new BABYLON.Color3(1, 1, 1);
+                scene.ambientColor = new BABYLON.Color3(0, 0, 0);
+
+                camera = new BABYLON.ArcRotateCamera("camera", alpha, beta, radius, BABYLON.Vector3.Zero(), scene);
+                camera.attachControl(canvas);
+                camera.wheelPrecision = 100.0;
+                camera.minZ = 0.1;
+                camera.maxZ = 1000;
+
+                var hdrTexture = new BABYLON.HDRCubeTexture(backgroundFileName, scene);
+
+                for (var i = 0; i < scene.meshes.length; i++) {
+                    var material = scene.meshes[i].material;
+                    if (material instanceof BABYLON.MultiMaterial) {
+                        for (var j = 0; j < material.subMaterials.length; j++) {
+                            var subMaterial = material.subMaterials[j];
+                            if (subMaterial instanceof BABYLON.PBRMaterial) {
+                                subMaterial.reflectionTexture = hdrTexture;
+                            }
+                        }
+                    }
+                }
+
+                var hdrSkybox = BABYLON.Mesh.CreateBox("hdrSkyBox", 1000.0, scene);
+                var hdrSkyboxMaterial = new BABYLON.PBRMaterial("skyBox", scene);
+                hdrSkyboxMaterial.backFaceCulling = false;
+                hdrSkyboxMaterial.reflectionTexture = hdrTexture.clone();
+                hdrSkyboxMaterial.reflectionTexture.coordinatesMode = BABYLON.Texture.SKYBOX_MODE;
+                hdrSkyboxMaterial.microSurface = 1.0;
+                hdrSkyboxMaterial.cameraExposure = 0.6;
+                hdrSkyboxMaterial.cameraContrast = 1.6;
+                hdrSkyboxMaterial.disableLighting = true;
+                hdrSkybox.material = hdrSkyboxMaterial;
+                hdrSkybox.infiniteDistance = true;
+
+                engine.runRenderLoop(function () {
+                    scene.render();
+                });
+            });
+        }
+
+        function onModelChange() {
+            var modelExtension = options.folder.indexOf("Binary") !== -1 ? ".glb" : ".gltf";
+            var backgroundExtension = "";
+
+            // maps to attribution for image files here per cc license guidelines
+            // first element must match background name in gui menu
+            var imageSources = {
+                "country": " ",

```

Comment:
country is an image from hdrlib.com (hard to paste link since I couldn't find a way to do it)
License: CC
Source: Openfootage
Source Link: Openfootage.net
Download: http://hdrlib.rat9.com/request_download.php?request_id=5418358